skypy/halo/tests/test_quenching.py

```diff
@@ -0,0 +1,29 @@
+import numpy as np
+from scipy import stats
+from collections import Counter
+
+from skypy.halo.quenching import environment_quenched, mass_quenched
+
+
+def test_environment_quenched():
+    # Test the quenching process follows a binomial distribution
+    n, p = 1000, 0.7
+    quenched = environment_quenched(n, p)
+    number_quenched = Counter(quenched)[0]
+
+    p_value = stats.binom_test(number_quenched, n=n, p=p,
+                               alternative='greater')
+    assert p_value > 0.05
+
+
+def test_mass_quenched():
+    # Test the quenching process follows a binomial distribution
```

Comment:
```suggestion
    # Test the quenching process follows a binomial distribution if the
    # logarithmic halo masses are symmetrical about the offset
```
Be clear that this test only works because the log-masses are symmetric about the offset